src/lib/customers/20minuten-de.json

```diff
@@ -0,0 +1,54 @@
+	"background": "20 Minuten ist die schweizweit grösste Pendlerzeitung und eine der meistbesuchten Webseiten der Schweiz. Es ist ein Medienportal mit täglichen Beiträgen aus dem Bereich News, Unterhaltung und Inspiration. 20 Minuten hatte 2019 eine grosse Migration ihrer Applikationen weg von der Legacy Infrastruktur und dem teilweise fast 20-jährigen Applikationscode geplant. Dabei ging es um eine Migration von bestehenden VMs auf eine moderne Kubernetes-basierte Containerinfrastruktur auf AWS. Das Backend sollte von Perl auf Go migriert und containerisiert werden. Das Frontend wurde ebenfalls von Perl gerendert und sollte in eine moderne Next.js Applikation umgewandelt werden. 20 Minuten hat täglich über eine Million Nutzer:innen. Dies macht eine Migration ohne Downtime besonders anspruchsvoll. Zusätzlich, war es wichtig, die Betriebskosten trotz dieses grossen Loads in einem überschaubaren Rahmen zu halten.",
+	"goal": "Das Ziel des Projekts war die Modernisierung des Setups der Applikation. Dabei sollten die Legacykomponenten und der frühere Code abgelöst werden, um das System stabiler und kosteneffizienter zu machen. AWS wurde als Infrastrukturprovider ausgewählt und die Applikation sollten in Go, Node.js und React Microservices auf einem EKS (Kubernetes) Cluster neu implementiert werden.",
+	"contribution": {
+		"intro": "bespinian hatte eine Schlüsselrolle in den folgenden Themen bezüglich der Applikationsmigration im Sinne von Refactoring / Re-architecting aus den 6 Migrationsstrategien von AWS:",
+		"topics": [
+			{
+				"title": "Infrastruktur auf AWS",
+				"body": "bespinian hat massgeblich zur Provisionierung der Infrastruktur auf AWS beigetragen. Dabei hatten wir eine Schlüsselrolle im Schreiben und Anwenden der entsprechenden Terraform-Module und im Aufsetzen der Kubernetes Cluster, auf denen die verschiedenen Microservices gehostet werden. Ein wichtiger Teil war dabei auch das korrekte Konfigurieren des Caching-Layers (Amazon CloudFront), um bei der grossen Nutzer:innenzahl von 20 Minuten die Kosten im überschaubaren Rahmen zu halten. bespinian hat viel von der technischen Umsetzung und Automatisierung, aber auch von der Kommunikation zwischen den Entwicklungs- und den Infrastruktur-Teams übernommen."
+			},
+			{
+				"title": "Kubernetes- und Container-Konfiguration",
+				"body": "Eine spannende Aufgabe war die Konfiguration des Kubernetes Clusters (auf Amazon EKS). Die Architektur sowie die Umsetzung davon trägt noch heute die Handschrift von bespinian. Es ging dabei um architektonische Entscheidungen, aber auch um die Umsetzung der Provisionierung und Konfiguration des Clusters über Terraform. Anschliessend ging es um die Automatisierung der Deployments der einzelnen Services über Helm und GitHub Actions. In all diesen Teilbereichen durfte bespinian mitwirken. Auch eine grosse Rolle bezüglich der Koordination der einzelnen Teams und Applikationen im Sinne einer DevOps Migration wurde uns zuteil."
```

Comment:
Replace `In all diesen Teilbereichen durfte bespinian mitwirken. Auch eine grosse Rolle bezüglich der Koordination der einzelnen Teams und Applikationen im Sinne einer DevOps Migration wurde uns zuteil.` by `bespinian spielte eine essenzielle Rolle in allen diesen Teilbereichen. Ferner wurden wir auch mit der Koordination der einzelnen Teams und Applikationen im Sinne einer DevOps Migration betraut.`